build_system.sh

```diff
@@ -24,18 +24,29 @@ SKIP_CHUNKS_IMAGE="$OUTPUT_DIR/system-skip-chunks.img"
 # Create temp dir if non-existent
 mkdir -p $BUILD_DIR $OUTPUT_DIR
 
-# Copy kernel modules over
+# Check if kernel modules are built
 if ! ls $OUTPUT_DIR/*.ko >/dev/null 2>&1; then
-  echo "kernel modules missing. run ./build_kernel.sh first"
+  echo "Kernel modules missing. Run ./build_kernel.sh first"
   exit 1
 fi
+
+# Copy kernel modules over only if updated - otherwise prevents proper caching later
```

Comment:
when does this break the caching?

Reply:
in the last steps of the Dockerfile where `COPY ./userspace/usr/comma/ /usr/$USERNAME/` is copied then if the kernel module are always copied, then also the docker build stages run again (even if the modules are exactly the same)

Reply:
fyi, this adds very little time (seconds) to the build, but also creates additional layers that will accumulate to the namespace docker cache

Reply:
let's remove it. these kinds of micro-optimizations tend not to be worth it